zoloto/coords.py

```diff
@@ -1,13 +1,107 @@
-from typing import NamedTuple
+from typing import Iterator, NamedTuple, Tuple
 
+from cached_property import cached_property
 from coordinates import spaced_coordinate
+from cv2 import Rodrigues
+from pyquaternion import Quaternion
 
 Coordinates = spaced_coordinate("Coordinates", "xy")
 
-Orientation = NamedTuple(
-    "Orientation", [("rot_x", float), ("rot_y", float), ("rot_z", float)]
-)
-
 ThreeDCoordinates = spaced_coordinate("ThreeDCoordinates", "xyz")
 
 Spherical = NamedTuple("Spherical", [("rot_x", float), ("rot_y", float), ("dist", int)])
+
+ThreeTuple = Tuple[float, float, float]
+RotationMatrix = Tuple[ThreeTuple, ThreeTuple, ThreeTuple]
```

Comment:
Would these be better defined as `NamedTuple`?

Reply:
Unlikely, its a rotation matrix, so values have no other name than their position in the matrix.